main/guides/README.md

```diff
@@ -0,0 +1 @@
+This folder contains various Agoric Programming Environment guides.
```

Comment:
We should change this text to something that is usable by a reader of the website, or delete this file altogether. Here's a screenshot of what it looks like:

<img width="920" alt="Screen Shot 2021-03-19 at 11 25 09 AM" src="https://user-images.githubusercontent.com/2441069/111826375-d9ccef80-88a5-11eb-9ae2-881928f0a48b.png">

How about a brief list of the guides here?